src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -34,6 +34,7 @@ import (
 )
 
 const (
+	failedOperatorRetry       = 10
```

Comment:
Consider adding a comment that 10 retries provide 5 minutes buffer for the operator to be in Failed status
The waitForOLMOperators is called with 30 seconds interval:
https://sourcegraph.com/github.com/openshift/assisted-installer@64bb7de193acba8112c0d632befb500863580538/-/blob/src/assisted_installer_controller/assisted_installer_controller.go?L436